package.json

```diff
@@ -46,14 +46,14 @@
     "csv": "^1.1.0",
     "debug": "2.2.0",
     "optimist": "0.6.1",
-    "sphere-node-sdk": "^1.16.2",
+    "sphere-node-sdk": "1.17.5",
```

Comment:
I think we should add the `^` back.

Caused the test to fail on my machine, I wonder how it was passing on CI 🤔